### PR TITLE
Fix venvlauncher

### PR DESCRIPTION
**Pull request by paulmon**
Fix test_venv failure on Windows arm32
@jslobodzian

### CODE REVIEW

File: Tools/winiot/sync_win_iot.py [hunk 1]
```diff
@@ -270,17 +266,40 @@ def main():
         print ("full layout selected")
         layout = FULL_LAYOUT
 
-    for t, s, p, c, altname in layout:
+    for t, s, p, c in layout:
```
Comment (jslobodzian) [inline Tools/winiot/sync_win_iot.py]: what kind of for expression is that?

Comment (chzawist) [reply Tools/winiot/sync_win_iot.py]: Inline deconstruction of a 4-element tuple. `layout` must be a sequence of 4-tuples for this to work.

It's basically the equivalent of
```
for each in layout:
    t = each[0]
    s = each[1]
    p = each[2]
    c = each[3]
```

Comment (paulmon) [reply Tools/winiot/sync_win_iot.py]: It iterates over each row in layout, assigning the row fields to the variables.  
This code was copied from make_zip.py

**Comment by paulmon** [comment]
dropped the readme file changes and force pushed

File: Tools/winiot/sync_win_iot.py [hunk 2]
```diff
@@ -141,6 +143,8 @@ def include_in_tools(p):
 FULL_LAYOUT_DEBUG = [
     ('/', '$source', 'python_d.exe', is_debug),
     ('/', '$source', 'pythonw_d.exe', is_debug),
+    ('Lib/', '$source', 'venvlauncher_d.exe', is_not_debug),
```
Comment (jslobodzian) [inline Tools/winiot/sync_win_iot.py]: is_debug...not is_not_debug.  This line and one below.

Comment (paulmon) [reply Tools/winiot/sync_win_iot.py]: fixed

**Pull request merged**